### ISSUE TITLE
Input file not in genbank format.

### ISSUE THREAD

**Issue by Wwwwwwwyc** [nextgenusfs/funannotate#895]
hi！
我在使用funannotate(1.8.14)进行基因预测时，遇到了问题。

这是我的输入命令：funannotate-docker predict -i genome.fasta -o predict -s "Candida auris"

但是在训练模型时，收到了以下错误：
Training Augustus using BUSCO gene models
size 120 is greater than the number of genes in file
/data1/home/user/predict/predict_misc/augustus.training.busco.gb. Aborting.

[Apr 06 01:00 PM]: AUGUSTUS training failed, check logfiles

我检查了logfile:
The necessary files for training candida_auris have been created.
Now, either run etraining or optimize_parameters.pl with --species=candida_auris.
etraining quickly estimates the parameters from a file with training genes.
optimize_augustus.pl alternates running etraining and augustus to find optimal metaparameters.

etraining: ERROR
	Input file not in genbank format.

请问我该如何解决？期待您的回复

**Comment by Wwwwwwwyc** [comment]
I used PASA prediction to try to bypass this link

for i in XXXXX ;do  funannotate-docker predict -i repeatmask/${i}.mask.fasta -o predict/${i} -s "XXX XXX" --transcript_evidence trinity.fasta --rna_bam ${i}.bam --min_training_models 10 --cpus 80 --pasa_gff train/training/funannotate_train.pasa.gff3 ;done




But it didn't work

[Apr 07 04:06 PM]: Running funannotate v1.8.14
[Apr 07 04:06 PM]: GeneMark not found and $GENEMARK_PATH environmental variable missing. Will skip GeneMark ab-initio prediction.
[Apr 07 04:06 PM]: Parsed training data, run ab-initio gene predictors as follows:
  Program        Training-Method
  augustus       pasa           
  codingquarry   rna-bam        
  glimmerhmm     pasa           
  snap           pasa           
[Apr 07 04:07 PM]: Loading genome assembly and parsing soft-masked repetitive sequences
[Apr 07 04:07 PM]: Genome loaded: 7 scaffolds; 12,249,773 bp; 29.80% repeats masked
[Apr 07 04:07 PM]: Existing transcript alignments found: predict/XXXXX/predict_misc/transcript_alignments.gff3
[Apr 07 04:07 PM]: Existing RNA-seq BAM hints found: predict/XXXXX/predict_misc/hints.BAM.gff
[Apr 07 04:07 PM]: Existing protein alignments found: predict/XXXXX/predict_misc/protein_alignments.gff3
[Apr 07 04:07 PM]: Filtering PASA data for suitable training set
[Apr 07 04:07 PM]: 1,081 of 3,186 models pass training parameters
[Apr 07 04:07 PM]: Training Augustus using PASA gene models
size 108 is greater than the number of genes in file
/data1/home/wangyc/C_auris_pangenome/represents_strain/predict/XXXXX/predict_misc/augustus.training.pasa.gb. Aborting.

[Apr 07 04:07 PM]: AUGUSTUS training failed, check logfiles

**Comment by nextgenusfs** [comment]
To test if it is your install or your data can you run `funannotate test -t busco --cpus n` and ensure those tests pass? If they pass okay then BUSCO is working properly, then its a question of what organism is this and if you are using appropriate models.

**Comment by Wwwwwwwyc** [comment]
Thank you for your reply. I solved the problem. The reason for the failure is contigID. I set the strain#contig1 format ID as required by the downstream command, but augustus does not recognize # as an ID gff2gb.pl. I changed the ID to strain_contig1 and funannotate ran successfully.
